File: Study/4. JAVA/2022.10.04_JAVA/src/test3/Circle.java
package test3;

public class Circle {
	private int r;
	private final double pi = 3.14;
	
	public void setR(int r) 
	{
		if(r <= 0)
		{
			System.out.println("¿À·ù");
			this.r = 0;
		}
		else
			this.r = r;
	}
	public int getR() {return this.r;}
	public double getPi() {return this.pi;}
	
	public double getArea() {return pi*r*r;}
	public double getRound() {return 2*pi*r;}
	
}

File: Study/2. C Sharp/2022.11.21_CSharp/SolT1_1/Circle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SolT1_1
{
    public class Circle
    {
        public int r { get; set; }
        protected double PI = 3.14;
        // protected :: Circle과 Circle 클래스를 상속한 클래스에서만 사용 가능


        //public Circle()
        //{
        //}
        public Circle(int r)
        {
            this.r = r;
        }

        public double getArea()
        {
            return r * r * PI;
        }
        public double getRound()
        {
            return 2 * r * PI;
        }
    }
}
